astropy/units/format/vounit.py

```diff
@@ -32,6 +32,7 @@ class VOUnit(generic.Generic):
     @staticmethod
     def _generate_unit_names():
         from ... import units as u
+        from ...units import deprecated as u_deprecated
```

Comment:
yep, that's why the tests were failing - in fact it's supposed to be *only* that